gulpfile.js

```diff
@@ -284,7 +284,7 @@ gulp.task('jsdoc', cb => {
  * Test
  */
 
-gulp.task('test', ['test:unit', 'test:a11y']);
+gulp.task('test', ['test:unit']);
```

Comment:
Yes, @hellobrian is in the process of finding another way for us to test for a117 as we're experiencing a lot of issues with the IBM one :-/